src/main/java/org/springframework/cloud/deployer/spi/kubernetes/DefaultContainerFactory.java

```diff
@@ -54,6 +53,7 @@
  * @author David Turanski
  * @author Chris Schaefer
  * @author Ilayaperumal Gopinathan
+ * @author Glenn Renfro
```

Comment:
Do you mind bumping the copyright as well?

Reply:
Sure.  What year is it? :-D